secrets/templates/bigmon.yaml

```diff
@@ -26,17 +26,17 @@ stringData:
 
   # Auth
   BIGMON_AUTH_PROVIDER_LIST: {{ join "," $auth_providers | quote}}
-  {{- range $provider, $value := .Values.bigmon.auth.providers}}
-  SOCIAL_AUTH_{{upper $provider}}_KEY: {{$value.key}}
-  SOCIAL_AUTH_{{upper $provider}}_SECRET: {{$value.secret}}
+  {{- range $provider, $v := .Values.bigmon.auth.providers }}
+  {{- range $kk, $vv := $v }}
+  SOCIAL_AUTH_{{ upper $provider }}_{{ upper $kk }}: {{ $vv }}
+  {{- end }}
   {{- end }}
-  SOCIAL_AUTH_INDIGOIAMOIDC_BASEPATH:  "{{ .Values.auth.indigoiam.basePath }}"
```

Comment:
Don't you need the basepath for indigo IAM? Will it be ok with only KEY and SECRET? I've seen you are including it at helm/bigmon/charts/main/sandbox/local.py.template

Reply:
It is set at

https://github.com/PanDAWMS/panda-k8s/blob/main/secrets/templates/bigmon.yaml#L31